web/src/main/java/ch/vorburger/minecraft/storeys/web/LocationToolListener.java

```diff
@@ -132,7 +131,7 @@ private void registerCondition(UUID player, String locationName, Pair<Location<W
             unregisterable.unregister();
         }
         ConditionService.ConditionServiceRegistration registration = conditionService.register(condition,
-                (Player p) -> eventBusSender.send(new JsonObject().put("event", name).put("playerUUID", p.getUniqueId().toString())));
+                (Player p) -> LOG.debug("TODO inform the player that someone was inside the region {}", p));
```

Comment:
is `LOG.debug()` visible with default Minecraft logging, or should we better make this an INFO or a WARN? Also, what we're really saying here is probably TODO.

```suggestion
                // TODO properly implement full support for this old feature in the new api-jvm
                (Player p) -> LOG.warn("TODO inform the player that someone was inside the region {}", p));
```